ci/ciimage/build.py

```diff
@@ -78,6 +78,10 @@ def gen_bashrc(self) -> None:
             if [ -f "$HOME/.cargo/env" ]; then
                 source "$HOME/.cargo/env"
             fi
+
+            if [ -f /etc/profile ]; then
+                source /etc/profile
+            fi
```

Comment:
Wow...

OpenSUSE fails to build because ci/ciimage/opensuse/install.sh uses `set -e` to run a series of sequential commands and fail if any of them fail.

And it also sources env_vars.sh which it expects to just contain env vars... but OpenSUSE's `/etc/profile` contains commands that can fail and aren't safe to run with `set -e`, which is pretty sketchy for a file that may get sourced by any random process in the history of ever.

Ultimately the issue is incredible quantities of total sketchiness.

```
#
# Call common progams from /bin or /usr/bin only
#
_path ()
{
    command -p ${1+"$@"}
}

#
# Initialize terminal
#
tty=`_path tty 2> /dev/null`
test $? -ne 0 && tty=""
```

Reply:
Worked around per IRC discussion...